### DISCUSSION TITLE
Graph(old) Legend names in 8.1.1

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#37823] [Dashboards, Panels & Visualizations]
I just upgraded from 7.5.7 to 8.1.1, here is my Graph(old). 

![image](https://user-images.githubusercontent.com/43188411/129163765-e998c0da-2c26-4485-a48f-c053762e5b8a.png)

```
SELECT $__timeGroup("date", $__interval, 0), 
COUNT("date"), id
....
GROUP BY id
```

I don't want 'count id' in legend names, only id is enough. It was working before 7.5.7. How to change the legend?

**Comment by torkelo** [comment]
can you change query so it read 
```
SELECT $__timeGroup("date", $__interval, 0), 
COUNT("date"), id as 'metric' 
....
GROUP BY id
```

**Reply by mertbakir** [reply]
We were using "tags" before 8, should we use 'metric' instead from now on?
It worked btw, ty.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
